### ISSUE TITLE
Building on M2 mac

### ISSUE THREAD

**Issue by alicedb2** [gridap/MiniQhull.jl#27]
Here's the error I get when trying to build MiniQhull. I'm using Julia 1.8.5 on macOS 13.2.1 on a 2022 M2 macbook air.

```
(MultivariateNormalCRP) pkg> build MiniQhull
    Building CMake ────→ `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/50a8b41d2c562fccd9ab841085fc7d1e2706da82/build.log`
    Building MiniQhull → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/e21881c219cc9de70ee990fabfd1f031aa4f414a/build.log`
ERROR: Error building `MiniQhull`: 
ERROR: LoadError: UndefVarError: artifact_dir not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:31
 [2] top-level scope
   @ ~/.julia/packages/MiniQhull/PLvbq/deps/build.jl:19
 [3] include(fname::String)
   @ Base.MainInclude ./client.jl:476
 [4] top-level scope
   @ none:5
in expression starting at /Users/alice/.julia/packages/MiniQhull/PLvbq/deps/build.jl:15

(MultivariateNormalCRP) pkg>
```

**Comment by jvellojin** [comment]
> Here's the error I get when trying to build MiniQhull. I'm using Julia 1.8.5 on macOS 13.2.1 on a 2022 M2 macbook air.
> 
> ```
> (MultivariateNormalCRP) pkg> build MiniQhull
>     Building CMake ────→ `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/50a8b41d2c562fccd9ab841085fc7d1e2706da82/build.log`
>     Building MiniQhull → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/e21881c219cc9de70ee990fabfd1f031aa4f414a/build.log`
> ERROR: Error building `MiniQhull`: 
> ERROR: LoadError: UndefVarError: artifact_dir not defined
> Stacktrace:
>  [1] getproperty(x::Module, f::Symbol)
>    @ Base ./Base.jl:31
>  [2] top-level scope
>    @ ~/.julia/packages/MiniQhull/PLvbq/deps/build.jl:19
>  [3] include(fname::String)
>    @ Base.MainInclude ./client.jl:476
>  [4] top-level scope
>    @ none:5
> in expression starting at /Users/alice/.julia/packages/MiniQhull/PLvbq/deps/build.jl:15
> 
> (MultivariateNormalCRP) pkg>
> ```

How you solved this?. I am getting the same error.

**Comment by alicedb2** [comment]
This issue was fixed in v0.4.0 with [#21](https://github.com/gridap/MiniQhull.jl/pull/21). In my case I didn't realize some other package in my environment was pinning MiniQhull to an earlier version.

**Comment by jvellojin** [comment]
> This issue was fixed in v0.4.0 with [#21](https://github.com/gridap/MiniQhull.jl/pull/21). In my case I didn't realize some other package in my environment was pinning MiniQhull to an earlier version.

Thanks for the response. How can I avoid those pinning? I'm on a 2023 MacBook Pro m2 max. The error is popping out when installing GridapEmbedded.

**Comment by fverdugo** [comment]
> The error is popping out when installing GridapEmbedded.

I see that GridapEmbedded (master branch) is already working with MiniQhull.jl 0.4.0 

https://github.com/gridap/GridapEmbedded.jl/blob/94dfa39920c9e462f9a8bfbde666073ef5e97a73/Project.toml#L23

Once we release a new version in GridapEmbedded, the problem should be fixed.

Regards,
Francesc
